.eslintrc.js

```diff
@@ -59,13 +59,14 @@ module.exports = {
         'no-restricted-syntax': 'off',
       },
     },
+    {
+      files: ['*.json'],
+      extends: 'plugin:@cypress/dev/general',
+    },
   ],
   rules: {
     'no-duplicate-imports': 'off',
     'import/no-duplicates': 'off',
-    '@typescript-eslint/no-duplicate-imports': [
```

Comment:
Does this mean duplicate imports are now allowed? I added a duplicate import to a file and `yarn lint` passed.

Reply:
weird the rule was renamed. I'll double check and make sure I implemented it correctly

Reply:
OK I see what happened. so the rule was deprecated for `import/no-duplicates` which we have set to `off` but we really should have it turned on. However, the rule comes from `import/no-duplicates` which was referenced in `@cypress/eslint-plugin-dev` but not installed or configured (likely resolved from the root of the monorepo). I added the dependency to the package and configured the plugin correctly, so the linting rules for duplicate imports should now work correctly. done in e6e262ff5253e4009590d13eae183fef7728edda